src/mono/System.Private.CoreLib/src/System/ModuleHandle.cs

```diff
@@ -71,7 +71,7 @@ public RuntimeTypeHandle ResolveTypeHandle(int typeToken, RuntimeTypeHandle[]? t
                 throw new ArgumentNullException(string.Empty, "Invalid handle");
             IntPtr res = RuntimeModule.ResolveTypeToken(value, typeToken, ptrs_from_handles(typeInstantiationContext), ptrs_from_handles(methodInstantiationContext), out _);
             if (res == IntPtr.Zero)
-                throw new TypeLoadException(string.Format("Could not load type '0x{0:x}' from assembly '0x{1:x}'", typeToken, value.ToInt64()));
+                throw new TypeLoadException(SR.Format("Could not load type '0x{0:x}' from assembly '0x{1:x}'", typeToken, value.ToInt64()));
```

Comment:
This should either use `string.Format` or the message should be moved into `Strings.resx` (for all cases in this file)